plugins/scaffolder-backend/sample-templates/react-ssr-template/{{cookiecutter.component_id}}/docs/index.md

```diff
@@ -0,0 +1,28 @@
+# {{ cookiecutter.component_id }}
+
+{{ cookiecutter.description }}
+
+## Getting started
+
+Start write your documentation by adding more markdown (.md) files to this folder (/docs) or replace the content in this file.
+
+## Table of Contents
+
+The Table of Contents on the right is generated automatically based on the hierarchy
+of headings. Only use one H1 (`#` in Markdown) per file.
+
+## Site navigation
+
+For new pages to appear in the left hand navigation you need edit the `mkdocs.yml`
+file in root of your repo. The navigation can also link out to other sites.
+
+Alternatively, if there is no `nav` section in `mkdocs.yml`, a navigation section
+will be created for you. However, you will not be able to use alternate titles for
+pages, or include links to other sites.
+
+Note that MkDocs uses `mkdocs.yml`, not `mkdocs.yaml`, although both appear to work.
+See also <https://www.mkdocs.org/user-guide/configuration/>.
```

Comment:
I struggled a bit when I wanted to create a "folding" navigation bar. I think we should link a very good example of `mkdocs.yml` here (if or when we have it).

Reply:
Good point! I think we can add more examples (on using our extensions etc as well) to this default docs content when it gets added to more templates. This is just a first step.

Reply:
Very soon we want this to be a optional step (but pre-selected) so you can "opt out" docs if you don't want it and at that point we can start adding more things.